packages/quantic/force-app/main/default/lwc/quanticTooltip/quanticTooltip.css

```diff
@@ -1,24 +1,57 @@
+:host {
+  --x-translation: var(--adapted-x-translation, 0px);
+  --max-width: var(--adapted-max-width, 20rem);
+}
+
 .tooltip__content {
   left: 50%;
-  transform: translate(-50%, -100%);
+  transition: transform 0.05s linear 0.05s, opacity 0.1s linear 0s, visibility 0.1s linear 0s;
+  z-index: 2;
+}
+
+.tooltip__content--positioned-above {
+  transform: translate(calc(-50% + var(--x-translation)), -100%);
   top: -0.7rem;
 }
 
+.tooltip__content--positioned-below {
+  transform: translate(calc(-50% + var(--x-translation)), 0.7rem);
+}
+
 .tooltip__content--visible {
   visibility: visible;
   opacity: 1;
 }
 
-.tooltip__content--light {
+.tooltip__content--light,
+.slds-popover_tooltip {
   width: max-content;
-  max-width: var(--slds-c-tooltip-sizing-max-width, var(--sds-c-tooltip-sizing-max-width, var(--lwc-sizeMedium, 20rem)));
 }
 
-.slds-nubbin_bottom-left:before,
-.slds-nubbin_bottom-left:after {
+.tooltip__arrow {
   left: 50%;
+  transform: translate(-50%, -100%);
+  transition: opacity 0.08s linear 0s, visibility 0.06s linear 0s;
```

Comment:
```suggestion
  transition: opacity opacity 0.1s linear 0s, visibility 0.1s linear 0s;
```
I think your two transitions should be the same delay so the tooltip arrow isn't visible before the content?

Reply:
I agree here, it would prevent this glitchy behaviour:
<img width="294" alt="image" src="https://github.com/coveo/ui-kit/assets/73316533/4e947f74-2540-45f1-9973-7ca17092b201">